queries/djot/highlights.scm

```diff
@@ -1,14 +1,28 @@
-(heading1) @markup.heading.1
+(heading) @markup.heading
 
-(heading2) @markup.heading.2
+((heading
+  (marker) @heading.marker) @markup.heading.1
+  (#eq? @heading.marker "# "))
```

Comment:
```suggestion
  (marker) @_heading.marker) @markup.heading.1
  (#eq? @_heading.marker "# "))
```

(We dropped that as an official capture (for color schemes).)

Reply:
Damn you're fast!